multi-llm-control-room/backend/app/plugins/crewai_plugin.py

```
@@ -0,0 +1,218 @@
+
+    model_adapter: Any
+    model_id: str
+
+    @property
+    def _llm_type(self) -> str:
+        return "control_room"
+
+    def _call(
+        self,
+        prompt: str,
+        stop: Optional[List[str]] = None,
```

Ruff (0.14.8)

Unused method argument: `stop`

(ARG002)